app/controllers/gigs.js

```diff
@@ -76,7 +76,7 @@ module.exports = function(app) {
         title: req.body.title,
         details: req.body.details,
         credits: parseInt(req.body.credits),
-        admin_task: req.body.admin_gig
+        admin_task: req.body.admin_gig ? true : false
```

Comment:
I think this might not be needed anymore, there was an issue parsing the boolean value on the service side, but I think that was resolved.